client-reactjs/src/components/application/actions/landingZoneFileUpload/LandingZoneUpload.jsx

```diff
@@ -1,38 +1,34 @@
 import React, {useState, useEffect} from 'react';
 import {useSelector} from "react-redux"
-import { Upload, Table, Select, message, TreeSelect, Button,Checkbox,Spin    } from 'antd';
-import { InboxOutlined, LoadingOutlined } from '@ant-design/icons';
+import { Upload, Table, Select, message, Button,Checkbox,Spin, Cascader, Tooltip,    } from 'antd';
+import { InboxOutlined, LoadingOutlined , CheckCircleOutlined, CloseCircleOutlined} from '@ant-design/icons';
 import { io } from "socket.io-client";
 import{LandingZoneUploadContainer, columns } from "./landingZoneUploadStyles";
 import {v4 as uuidv4} from 'uuid';
 import { authHeader, handleError } from "../../../common/AuthHeader";
 import { useHistory } from 'react-router';
-import { applicationReducer } from '../../../../redux/reducers/ApplicationReducer';
 
 function LandingZoneUpload() {
   const [files, setFiles] = useState([]);
   const [socket, setSocket] = useState(null);
   const [tableData, setTableData] = useState([]);
-  const [destinationFolder, setDestinationFolder] = useState("");
+  const [uploadAttempted, setUploadAttempted] = useState([]);
   const [cluster, setCluster] = useState(null);
-  const [dropzones, setDropZones] = useState([]);
-  const [selectedDropZone, setSelectedDropZone]= useState(null)
-  const [machine, setMachine] = useState(null);
-  const [directory, setDirectory] = useState(null);
-  const [currentDirectoryFiles, setCurrentDirectoryFiles] = useState([])
-  const [treeData, setTreeData] = useState([]);
+  const [machine, setMachine] = useState(null); //ip address of dropzone
+  const [currentDirectoryFiles, setCurrentDirectoryFiles] = useState([]);
+  const [pathToAsset, setPathToAsset] = useState('')
   const [overWriteFiles, setOverWriteFiles] = useState(false);
   const [uploading, setUploading] = useState(false);
-  const [successItem, setSuccessItem] = useState(null);
+  const [options, setOptions] = useState([]);
   const authReducer = useSelector(state => state.authReducer);
   const clusters = useSelector(state => state.applicationReducer.clusters);
-  const app = useSelector(state => state.applicationReducer)
-  const devURL = 'http://localhost:3000/landingZoneFileUpload';
+  const devURL = `${process.env.REACT_APP_PROXY_URL}/landingZoneFileUpload`;
   const prodURL  = '/landingZoneFileUpload';
   const { Dragger } = Upload;
   const { Option,  } = Select;
   const history = useHistory();
 
+
 useEffect(() => {
     // Socket io connection
     if(process.env.NODE_ENV === "development"){
```

Comment:
U basically changing one var so maybe it's better to have it this way
` 
const url = process.env.NODE_ENV === "development" ?  devURL : prodURL ;
    const socket = io(url , {
        transports: ["websocket"],
        auth : {  token : authReducer?.user.token }
        });
        setSocket(socket);
 `